types/priv_validator.go

```diff
@@ -225,7 +227,14 @@ func (privVal *PrivValidatorFS) save() {
 		// `@; BOOM!!!
 		cmn.PanicCrisis(err)
 	}
-	err = cmn.WriteFileAtomic(privVal.filePath, jsonBytes, 0600)
+	outFile := privVal.filePath
+	// We need to ensure that the parent directory exists.
+	// See Issue https://github.com/tendermint/tendermint/issues/1290
+	dir := filepath.Dir(outFile)
+	if err := os.MkdirAll(dir, 0700); err != nil {
+		cmn.PanicCrisis(err)
```

Comment:
AFAIK `Panic` methods are deprecated, we use blank `panic`s now.

Reply:
I see, thank you. I was using the surrounding code as the guide. However, I'll change that up.